db/migrations/20251112041817_test_runs_update_up.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+CREATE TABLE test_run_results (
+    id UUID PRIMARY KEY,
+    test_run_id UUID NOT NULL REFERENCES test_runs(id) ON DELETE CASCADE,
+    test_case_id UUID NOT NULL REFERENCES test_cases(id) ON DELETE CASCADE,
+    status TEXT NOT NULL CHECK (status IN ('passed', 'failed', 'skipped', 'blocked')),
+    result TEXT NOT NULL,
+    notes TEXT,
+    executed_by INT NOT NULL,
+    executed_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT now(),
+    created_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT now()
+);
+
+-- +goose Down
+DROP TABLE test_run_results;
```

Comment:
How is this consolidated with the fact that we already store results in `test_runs`? seems like a duplication to me